src/protectionmarketlist.schema.json

```diff
@@ -0,0 +1,441 @@
+          "$ref": "#/definitions/DivestScript"
+        },
+        "invest": {
+          "$ref": "#/definitions/InvestScript"
+        },
+        "strategy": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StrategyLeg"
+          }
+        },
+        "tokens": {
+          "type": "object",
+          "properties": {
+            "input": {
+              "type": "array",
+              "description": "Array of one or more tokens a user must have to invest in this opportunity",
+              "items": {
+                "$ref": "#/definitions/TokenInfo"
+              }
+            },
+            "receipt": {
+              "type": "array",
+              "description": "Array of one or more tokens a user will receive after investing in this opportunity",
+              "items": {
+                "$ref": "#/definitions/TokenInfo"
+              }
+            },
+            "rewards": {
+              "type": "array",
+              "description": "Array of one or more tokens a user will receive as a reward for investing in this opportunity",
+              "items": {
+                "$ref": "#/definitions/TokenInfo"
+              }
+            }
+          }
+        }
+      }
+    },
+    "ProtectionMarketInfo": {
+      "type": "object",
+      "description": "Metadata for a single market in a protection market list",
+      "additionalProperties": false,
+      "properties": {
+        "chainId": {
+          "type": "integer",
+          "description": "The chain ID of the Ethereum network where this token is deployed",
+          "minimum": 1,
+          "examples": [1, 42]
+        },
+        "address": {
+          "type": "string",
+          "description": "The checksummed address of the protection market on the specified chain ID",
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "examples": ["0xdBDF2fC3Af896e18f2A9DC58883d12484202b57E"]
+        },
+        "decimals": {
+          "type": "integer",
+          "description": "The number of decimals the market's receipt token has",
+          "minimum": 0,
+          "maximum": 255,
+          "examples": [18]
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the protection market",
+          "minLength": 1,
+          "maxLength": 100,
+          "pattern": "^[ \\w.'+\\-%\\/À-ÖØ-öø-ÿ:]+$",
+          "examples": ["Cozy USD Coin"]
+        },
+        "symbol": {
+          "type": "string",
+          "description": "The symbol for the protection market; must be alphanumeric",
+          "pattern": "^[a-zA-Z0-9+\\-%\\/\\$]+$",
+          "minLength": 1,
+          "maxLength": 100,
+          "examples": ["cozyUSDC"]
+        },
+        "logoURI": {
+          "type": "string",
+          "description": "A URI to the protection market's logo asset; if not set, interface will attempt to find a logo based on the token address; suggest SVG or PNG of size 64x64",
```

Comment:
How about 128x128? 

Will be better for hi-res screens :)

Reply:
Hm, yea I wonder why Uniswap chose 64x64 and 256x256 as the two options. Either way, the sizes in the descriptions are recommendations since they can't be enforced. But 256x256 sounds good to me, so updated in 42c55ff3364adc13231e658c49aaf05d36cc6430

Reply:
Hm, yea I wonder why Uniswap chose 64x64 and 256x256 as the two options. Either way, the sizes in the descriptions are recommendations since they can't be enforced. But 256x256 sounds good to me, so updated in 42c55ff3364adc13231e658c49aaf05d36cc6430